src/lib.rs

```diff
@@ -517,6 +531,25 @@ fn pan_orbit_camera(
         }
 
         // 4 - Update the camera's transform based on current values
+        pan_orbit.radius = pan_orbit.radius.map(|radius| {
+            if pan_orbit.target_radius != radius {
+                // Radius has been changed by zooming
+                has_moved = true;
+                // Smoothing for line-based scrolling
+                let t = 1.0 - pan_orbit.zoom_smoothness;
+                let mut next_radius = radius.lerp(&pan_orbit.target_radius, &t);
+                if approx_equal(next_radius, pan_orbit.target_radius) {
+                    next_radius = pan_orbit.target_radius
+                }
+                if let Projection::Orthographic(ref mut p) = *projection {
+                    p.scale = next_radius;
+                }
```

Comment:
As stated in other PR, you need to do this for pixel scrolling too

Reply:
I added the values of both line scrolling and pixel scrolling to the target_radius.

Reply:
> As stated in other PR, you need to do this for pixel scrolling too

I added the values of both line scrolling and pixel scrolling to the target_radius.